src/components/routeComponents/memberRegistrationCorrectionForm/AssignLevelAndClassroomLink.jsx

```diff
@@ -0,0 +1,68 @@
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const LinkStyled = styled.a`
+	color: #f9570a;
+	font-weight: 500;
+	font-size: 14px;
+	line-height: 21px;
+	border-bottom: 1px solid #f9570a;
+	&:hover {
+		color: darkorange;
+		cursor: pointer;
+	}
+`;
+
+const AssignLevelAndClassroomLink = ({
+	constants,
+	onClickShowAssignLevelAndClassroomModal,
+	member,
+	isLevelAndClassroomValue,
+}) => {
+	const { ASSIGN_LEVEL_AND_CLASSROOM, ASSIGNED_LEVEL, LEVEL, CLASSROOM } = constants;
+	if (isLevelAndClassroomValue) {
```

Comment:
Why do we need this condition ?
```suggestion
	if (isLevelAndClassroomValue) {
```